jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -1556,19 +1557,14 @@ ecma_string_trim (const ecma_string_t *string_p) /**< pointer to an ecma string
 {
   ecma_string_t *ret_string_p;
 
-  lit_utf8_size_t buffer_size = ecma_string_get_size (string_p);
+  ECMA_STRING_TO_UTF8_STRING (string_p, utf8_str_p);
 
-  if (buffer_size > 0)
+  if (utf8_str_p_size > 0)
```

Comment:
This is inside `ECMA_STRING_TO_UTF8_STRING (string_p, utf8_str_p);`. What is the problem with this? Is the problem that the deceleration is not visible?